code/modules/mob/inventory.dm

```diff
@@ -17,6 +17,51 @@
 		return 1
 	return 0
 
+
+/mob/proc/get_proper_equip_name(obj/item/W as obj,slot)
+
+	var/selfverb = "You equip the [W]."
+	var/otherverb = "[src] equips the [W]."
+	var/mob/living/carbon/human/srchuman = src
+
+	switch(slot)
+		if(slot_l_store)
+			selfverb = "You place \the [W] in your left pocket."
+			otherverb = "[src] stuffs \the [W] in their left pocket."
+		if(slot_r_store)
+			selfverb = "You place \the [W] in your right pocket."
+			otherverb = "[src] stuffs \the [W] in their right pocket."
+		if(slot_back)
+			selfverb = "You hoist \the [W] onto your back."
+			otherverb = "[src] hoists the [W] onto their back."
+		if(slot_wear_id)
+			selfverb = "You place \the [W] on your uniform."
+			otherverb = "[src] places \the [W] on your uniform."
+		if(slot_head)
+			selfverb = "You place \the [W] on your head."
+			otherverb = "[src] places \the [W] on their head."
+		if(slot_shoes)
+			selfverb = "You slip on \the [W]."
+			otherverb = "[src] slips on \the [W]."
+		if(slot_gloves)
+			if(istype(srchuman) && (!srchuman.l_hand || srchuman.r_hand))
+				selfverb = "You slide your hand into \the [W]."

```

Comment:
wouldn't this go like "the the W" due to how Byond handles shit?

Reply:
Doesn't do this for me.